addons/survey/tests/test_survey.py

```diff
@@ -475,3 +477,57 @@ def test_get_pages_and_questions_to_show(self):
         returned_questions_and_pages = my_survey._get_pages_and_questions_to_show()
 
         self.assertEqual(question_and_page_ids - invalid_records, returned_questions_and_pages)
+
+    @users('survey_manager')
+    def test_allowed_triggering_question_ids(self):
+        # Create 2 surveys, each with 3 questions, each with 2 suggested answers
+        survey_1, survey_2 = self.env['survey.survey'].create([
+            {'title': 'Test Survey 1'},
+            {'title': 'Test Survey 2'}
+        ])
+        self.env['survey.question'].create([
+            {
+                'survey_id': survey.id,
+                'title': f'Question {jdx}',
+                'question_type': 'simple_choice',
+                'suggested_answer_ids': [
+                    Command.create({
+                        'value': f'Answer {kdx}',
+                    }) for kdx in range(2)],
+            }
+            for jdx in range(3)
+            for survey in (survey_1, survey_2)
```

Comment:
"for survey_id in (survey_1 + survey_2).ids" ?

Does not really matter actually, but I like to use recordsets when possible ^^